content/libraries.md

```diff
@@ -6,8 +6,12 @@ sidebar:
     - header: Official Libraries
     - text: Python
       href: "#python"
+    - text: PHP
+      href: "#php"
     - text: Ruby
       href: "#ruby"
+    - text; JAVA
+      href: "#java"
     - text: Go
       href: "#go"
     - text: C#
```

Comment:
Agreed. Merge this first, then do another PR to alphabetize or do you want to alphabetize in this PR? This is good to merge if you wanted to do the former (merge, then new PR to alphabetize)